src/font.rs

```diff
@@ -200,16 +201,16 @@ impl Font {
     ///
     /// let datareq = DataRequest::none().lib(true);
     ///
-    /// let ufo = Font::load_requested_data("path/to/font.ufo", &datareq).expect("failed to load");
+    /// let ufo = Font::load_requested_data("path/to/font.ufo", datareq).expect("failed to load");
     /// ```
     pub fn load_requested_data(
         path: impl AsRef<Path>,
-        request: &DataRequest,
+        request: DataRequest,
```

Comment:
wouldn't this DataRequest -> &DataRequest be a breaking change?

Reply:
nevermind, norad hasn't reached 1.0 yet so it should be ok

Reply:
it is a breaking change, yes, and it will be made in the bump from 0.7 to 0.8.

Reply:
Another breaking change is the removal of the public `layers` field in `DataRequest`.